esx_service/vmdk_ops.py

```diff
@@ -209,6 +213,91 @@ def make_create_cmd(opts, vmdk_path):
         return "{0} -d {1} -c {2} {3}".format(VMDK_CREATE_CMD, disk_format, size, vmdk_path)
 
 
+def cloneVMDK(vm_name, vmdk_path, opts={}, vm_uuid=None, vm_datastore=None):
```

Comment:
both new params seem excessive. vm_name should be enough got find uuid, and vmdk_path should be enough to find data store. It's OK to have redundant params to save on extra search, I just want to make sure it's a deliberate choice ?